bash_completionsV2.go

```diff
@@ -224,20 +225,28 @@ __%[1]s_handle_completion_types() {
         # completions at once on the command-line we must remove the descriptions.
         # https://github.com/spf13/cobra/issues/1508
         local tab=$'\t' comp
-        while IFS='' read -r comp; do
+        for comp in "${completions[@]}"; do
             [[ -z $comp ]] && continue
             # Strip any description
             comp=${comp%%%%$tab*}
             # Only consider the completions that match
             if [[ $comp == "$cur"* ]]; then
                 COMPREPLY+=("$comp")
-            fi
-        done < <(printf "%%s\n" "${completions[@]}")
+           fi
+        done
+
+        __%[1]s_escape_compreply
+        ;;
+
+    63)
+        # Type: Listing completions after successive tabs
+        __%[1]s_handle_standard_completion_case
```

Comment:
I'm not escaping here so that the list of results on multiple tabs doesn't include any extra escape characters...is that the behavior you're expecting?

Reply:
Could you give an example of the behaviour you are referring too?

Reply:
`testprog prefix special-chars bash<tab><tab>`

The list of options it reports is not escaped (i.e. `bash4>redirect`), but if you were to do `bash4<tab>` you'd get the escaped version (`bash4\>redirect`)